Sources/BraintreePayPal/BTPayPalAnalytics.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class BTPayPalAnalytics {
```

Comment:
This is a bit hard to read what all of the tokenize/browser-presentation/etc events are. I think something like this would clean this file up a bit:
```swift
class BTPayPalAnalytics {

    // MARK: - Tokenize Request Events
    
    <all_tokenize_events>

    // MARK: - Browser Login Events
    
    <all_browser_login_events>

    // MARK: - Browser Presentation Events
    
    <all_browser_presentation_events>
}
```
This will also make it easier for us to ensure that we are keeping the naming of the events consistent